src/utils/__test__/NoteUtils.test.ts

```diff
@@ -6,15 +6,15 @@ describe("NoteUtils", () => {
   });
 
   it("no assignee note returns null", () => {
```

Comment:
A bit confused by what this test was/is testing. I can understand why we'd want the following assertion to be null, so perhaps we just need a better test name.

```javascript
expect(NoteUtils.getFollowUp([troubleticket], "assignee")).toBeNull();
```

Reply:
this is just testing that `null` is returned when the notes array does not contain an assignee (now field office) note. changed the message to `"it returns null when there are no notes with subtype fieldoffice"`